Hippo.Web/ClientApp/src/components/Account/ActiveAccounts.tsx

```diff
@@ -17,7 +17,7 @@ import { HipTable } from "../../Shared/Table/HipTable";
 import HipButton from "../../Shared/HipComponents/HipButton";
 
 export const ActiveAccounts = () => {
-  const [notification, setNotification] = usePromiseNotification();
+  const [, setNotification] = usePromiseNotification();
```

Comment:
use `_` to indicated unused param, like `const [_, setNotification] = usePromiseNotification();`

Reply:
Okay, but eslint gives a `no-unused-vars` warning for `_`.